modules/rosa-hcp-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-cluster-creation_{context}"]
+= Step Four - Cluster Creator: Creating your cluster in a shared VPC
+To create a cluster in a shared VPC, complete the following steps.
+
+[NOTE]
+====
+Installing a cluster in a shared VPC is supported only for OpenShift 4.17.9 and later and all future 4.y-streams.
+====
+
+image::372_OpenShift_on_AWS_persona_worflows_0923_4.png[]
+.Prerequisites
+
+* You have the hosted zone IDs from the *VPC Owner*.
+* You have the AWS region from the *VPC Owner*.
+* You have the subnet IDs from the *VPC Owner*.
+* You have the `Route 53 role` ARN from the *VPC Owner*.
+* You have the `VPC endpoint role` ARN from the *VPC Owner*.
+
+.Procedure
+* In a terminal, enter the following command to create the shared VPC:
```

Comment:
Should this be a numbered list instead of a bulleted list to match the rest of the document?

Reply:
Our writing guidelines state that one step procedures should be bullets.

Reply:
Got it, that makes sense.